recipes/libsodium/1.0.18/conanfile.py

```diff
@@ -115,6 +115,17 @@ def _build_autotools_darwin(self, configure_args):
         host_arch = "%s-apple-%s" % (self.settings.arch, os)
         configure_args.append("--host=%s" % host_arch)
         self._build_autotools_impl(configure_args)
+        
+    def _build_autotools_neutrino(self, configure_args):
```

Comment:
I was expecting conan to deduce host correctly for QNX.
if it's still not the case, please report an issue to the conan repository.
for the time being, it's okay to have this code in recipe, but it doesn't stale well to have it in every recipe...
so it's much better if conan deduces the host for QNX correctly.

Reply:
opened issue at https://github.com/conan-io/conan/issues/6461